backend/Origam.Rule/RuleEngine.cs

```diff
@@ -772,25 +772,13 @@ private bool ProcessRulesFromQueue(DataRow rowChanged, IDataDocument data, DataS
 		{
 			return ProcessRulesInternal(rowChanged, data, null, ruleSet, columnsChanged, true);
 		}
-
-		private Queue _ruleQueue = new Queue();
+		
+		private IndexedRuleQueue _ruleQueue = new ();
 		private Hashtable _ruleColumnChanges = new Hashtable();
 
 		private bool IsEntryInQueue(DataRow rowChanged, DataStructureRuleSet ruleSet)
```

Comment:
This method is now redundant, I guess.

Reply:
The _ruleQueue is private and it should probably stay that way. So I would keep the method.